gatsby-config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	// pathPrefix: `/slides`,
+	plugins: [
+		{
+			resolve: 'gatsby-theme-mdx-deck',
+			options: {
+				// source directory
+				contentPath: './slides',
+				// base path for routes generate by this theme
+				basePath: '/slides',
+			},
+		},
+		{
+			resolve: 'gatsby-plugin-compile-es6-packages',
```

Comment:
That's where you end up when you try to do zero config 🤦

Reply:
🤷🏻‍♂️